ZocBuild.Database/packages.config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Bcl" version="1.1.8" targetFramework="net40" />
+  <package id="Microsoft.Bcl" version="1.1.8" targetFramework="net40" requireReinstallation="True" />
   <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net40" />
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net40" />
-  <package id="System.IO.Abstractions" version="1.4.0.76" targetFramework="net45" />
+  <package id="System.IO.Abstractions" version="2.0.0.108" targetFramework="net45" />

```

Comment:
You should update the `<dependencies>` sections of the [nuspec](https://github.com/ZocDoc/ZocBuild.Database/blob/AddMoreLogging/ZocBuild.Database/ZocBuild.Database.nuspec) file.